### PR TITLE
Updated freeport guildmaster turning war cler

### PR DESCRIPTION
**Pull request by MATTG-ZEROFILL**

### CODE REVIEW

**Comment by MATTG-ZEROFILL** [comment]
Pull request should be titled turn-in, not "turning". Some additional missing quest checks were copied over from the new Freeport to the old as part of the newbie turn in fixes.

File: freportn/Eestyana_Naestra.lua [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+		e.other:SummonItem(51121); -- Item: Tradeskill Basics : Volume I
+	elseif(e.message:findi("second book")) then
+		e.self:Say("Here is the second volume of the book you requested, may it serve you well!");
+		e.other:SummonItem(51122); -- Item: Tradeskill Basics : Volume II
+	end
+end
+
+function event_enter(e)
+	if(e.other:HasItem(18735) == true) then
+		e.other:Message(15,"As you bask in the majesty of the Temple of Marr, an important looking individual, clad in shining metal armor turns to greet you. 'I can spot a new recruit a mile a way. Welcome to the Temple of Marr. I am Eastyana Naestra. Read the note in your inventory and hand it to me to begin your training. I wish you the best of luck.'");
```
Comment (fryguy503) [inline freportn/Eestyana_Naestra.lua:18]: I believe for this to properly function, you need to define a proximity: e.g.

function event_spawn(e)
	local xloc = e.self:GetX();
	local yloc = e.self:GetY();
	eq.set_proximity(xloc - 50, xloc + 50, yloc - 50, yloc + 50);
end

File: freportw/Cain_Darkmoore.lua [hunk 2]
```diff
@@ -1,18 +1,30 @@
...
+	elseif(e.message:findi("green and blue")) then
+		if(e.other:Class() == "Warrior") then
 			e.self:Say("The green and blue was a test for all skilled warriors. All one needed do was return to me four orc legionnaire shoulderpads. Two from the loathsome green skinned Deathfist Clan and two from the vile blue skinned Crushbone Clan. But we have no more bunker battle blades to spare. We must prepare for war!!");
 		end
 	end
 end
 
+function event_enter(e)
+	if(e.other:HasItem(18748) == true) then
+		e.other:Message(15,"A deep booming voice breaks the silence of the room as you get your bearings. 'Step forward young recruit. I am Cain Darkmoore, Guild Master of the Army of Steel. Read the note in your inventory and hand it to me when you are ready to begin your training. Of course if you are to afraid, I saw an Elf handing out flutes at the bards' Guild this morning.'");
```
Comment (fryguy503) [inline freportw/Cain_Darkmoore.lua:24]: Same here, need to define a proximity for the event_enter

**Comment by MATTG-ZEROFILL** [comment]
Yes of course, thanks! Missed the copy of those parts from the updated Freeport script versions. Will splice in.

**Pull request closed without merging**